src/hmrc-design-patterns/addresses/index.njk

```diff
@@ -136,32 +141,47 @@ Make sure errors follow the <a class="govuk-link" href="https://design-system.se
 </p>
 
 <p class="govuk-body">
-  If there is a problem with the postcode after ignoring any unwanted characters:
+  Only show these errors if there is a problem with the postcode after ignoring any unwanted characters.
 </p>
 
-<h4 class="govuk-heading-s" id="address-wrong-format">
-  If the postcode is in the wrong format
+<h4 class="govuk-body" id="address-wrong-format">
```

Comment:
Sorry missed this from the initial review... If we're styling these like a paragraph and the fact that they've now got a colon I think this should just be a paragraph element. Same for the rest below.